test/pythonml/test_model_flavors.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import importlib
+import os
+import shutil
+import unittest
+from parameterized import parameterized
+from fixtures.unittest_baseclass import BaseUnitTest
+
+
+
+class TestModelFlavors(BaseUnitTest):
+    """
+    This class performs unit tests for the model flavors
+    """
+
+    def check_pass_conditions_training(self, category):
+        import settings
+        importlib.reload(settings)
+        with settings.context as context:
+            if category == 'regression':
+                rmse = context.load('RMSE')
+                assert(rmse <= 2 * 10)
```

Comment:
Just do `2e1` or `20` here, unless there's some reason this multiplication needs to happen at runtime